### PR TITLE
Remove redundant test utils

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Remove redundant test utils](https://trello.com/c/2W5dQ53n/457-remove-redundant-test-utils)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8607/

File: package-lock.json [hunk 1]
```diff
@@ -1223,6 +1223,7 @@
       "requires": {
         "anymatch": "1.3.2",
         "async-each": "1.0.1",
+        "fsevents": "1.1.3",
```
Comment (hatched) [inline package-lock.json]: dueling fsevents - @frankban removes it, and you add it :)

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**